### PR TITLE
Community detection online documentation

### PR DESCRIPTION
**Pull request by antepusic**
Documentation for MAGE’s dynamic community detection query module (memgraph/mage#66).

### CODE REVIEW

File: mage/query-modules/cpp/dynamic_community_detection.md [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+## Procedures
+
+Performs dynamic community detection using the LabelRankT algorithm.
+
+The default values of the `exponent` and `min_value` parameters are not universally applicable,
+and the actual values should be tailored to the graph in question.
+This especially goes for the `min_value` parameter. With the default ***1/10*** value, vertices of degree greater than 10 
+are at risk of not being assigned to any community and the user should check if that is the case.
+
+### `detect(directed, weighted, similarity_threshold, exponent, min_value, weight_property, w_selfloop, max_iterations, max_updates)`
```
[the remark this answers is not in the document]
Comment (antepusic) [reply mage/query-modules/cpp/dynamic_community_detection.md]: You were right! Method renamed to `set()`.

**Review by jmatak — APPROVED** [review]

File: mage/overview.md [hunk 2]
```diff
@@ -57,4 +57,5 @@ the table below which will reveal MAGE compatibility with Memgraph versions.
...
 | [bridges](/mage/query-modules/cpp/bridges)                                         | C++    | A bridge is an edge, which when deleted, increases the number of connected components. The goal of this algorithm is to detect edges that are bridges in a graph.                                                                 |
 | [betweenness centrality](/mage/query-modules/cpp/betweenness-centrality)           | C++    | The betweenness centrality of a node is defined as the sum of the of all-pairs shortest paths that pass through the node divided by the number of all-pairs shortest paths in the graph. The algorithm has O(nm) time complexity. |
+| [dynamic_community_detection](/mage/query-modules/cpp/dynamic-community-detection) | C++    | Dynamic community detection algorithm suitable for large-scale graphs based upon label propagation. Runs in *O(m)* time and has *O(mn)* space complexity.                                                                         |
```
Comment (antepusic) [inline mage/overview.md:60]:
```suggestion
| [community_detection_online](/mage/query-modules/cpp/community-detection-online) | C++    | Online community detection algorithm suitable for large-scale graphs based upon label propagation. Runs in *O(m)* time and has *O(mn)* space complexity.                                                                         |
```

**Review by antepusic — COMMENTED** [review]
I left a fix suggestion.

File: mage/installation.mdx [hunk 3]
```diff
@@ -0,0 +1,119 @@
+---
+id: installation
+title: How to install MAGE?
+sidebar_label: Installation
+---
```
Comment (antoniofilipovic) [inline mage/installation.mdx:5]: I think this file needs to be removed, we already have `mage/installation.md `in the master branch.

**Review by g-despot — APPROVED** [review]

**Pull request merged**